source/templates.tex

```diff
@@ -6978,8 +6977,8 @@
 \pnum
 \begin{note}
 Because the explicit template argument list follows the function
-template name, and because conversion member function templates and
-constructor member function templates are called without using a
+template name, and because
+constructor member function templates\iref{class.ctor} are named without using a
```

Comment:
Is the change "are called" -> "are named" here intentional? I think it's clear that (for example) the expression `A()` calls a constructor without using a function name, and more generally, there is no syntax to call a constructor using a function name. However, it's not clear to me that `template<typename T> A::A(T t)` names the constructor without using a function name -- in that context, `A::A` names the constructor, and the constructor is a function, so arguably `A::A` is a function name.

Reply:
[temp.arg.explicit] p2 lists the circumstances under which an explicit template argument list can be supplied for a function template. "A function is called" is just one of five options. I agree that "named" is bad term here, though, because we use "naming the constructor" when we want to weasel-word around the fact that constructors don't have names to start with. p1 talks about "function template name", and the note wants to highlight that a strict reading of "name" is required here (and thus constructors can't have template argument lists).